rest/data/patch.go

```diff
@@ -286,6 +286,30 @@ func (p *DBPatchConnector) GetPatchRawPatches(patchID string) (map[string]string
 		}
 	}
 
+	// set the patch status to the collective status between the parent and child patches
+	if patchDoc.IsParent() {
+		allStatuses := []string{patchDoc.Status}
+		for _, childPatchId := range patchDoc.Triggers.ChildPatches {
+			cp, err := patch.FindOneId(childPatchId)
+			if err != nil {
+				return nil, gimlet.ErrorResponse{
```

Comment:
nit: you can use gimlet.MakeJSONInternalErrorResponder to simplify this. The long form is only really necessary with unusual status codes (such as status not found).

Reply:
gimlet.MakeJSONInternalErrorResponder returns a `gimlet.Responder` type, while gimlet.ErrorResponse returns an `error `type, which is what GetPatchRawPatches returns.